.changeset/khaki-camels-cheer.md

```diff
@@ -0,0 +1,9 @@
+---
+"@lux-design-system/components-react": major
+---
+
+In deze commit:
+
+- Nieuw component: LuxDocument
+- Nieuw component: LuxParagraph
+  - Wijziging t.o.v. Utrecht Paragraph: properties `lead` en `small` vervangen voor `appearance="lead|small"`
```

Comment:
Deze regel mag hier weg. Het is voor de gebruiker niet interessant welk component we hiervoor gebruiken of hoe we daarvan afwijken.